$NetBSD: patch-apps_CMakeLists.txt,v 1.5 2015/01/07 14:44:34 makoto Exp $

(1) Avoid ${WRKSRC} included in RPATH.
(2) WARN: PLIST:105: Man pages should be installed into man/, not share/man/.

--- apps/CMakeLists.txt.orig	2014-11-07 20:03:26.000000000 +0900
+++ apps/CMakeLists.txt	2014-11-19 22:36:36.000000000 +0900
@@ -120,8 +120,8 @@ if(APPLE)
 endif()
 
 if(NOT WIN32) #rpath setup
-	SET(CMAKE_SKIP_BUILD_RPATH  FALSE)
-	SET(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE) 
+	SET(CMAKE_SKIP_BUILD_RPATH TRUE)
+	SET(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE) 
 	SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 	SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
 endif()
@@ -149,7 +149,7 @@ if(NOT APPLE AND NOT WIN32)
 	install(TARGETS tqsl DESTINATION bin)
 	install(DIRECTORY help DESTINATION share/TrustedQSL)
 	install(FILES ca-bundle.crt DESTINATION share/TrustedQSL)
-	install(FILES tqsl.5 DESTINATION share/man/man5)
+	install(FILES tqsl.5 DESTINATION man/man5)
 	install(FILES icons/key48.png DESTINATION share/pixmaps RENAME TrustedQSL.png)
 	install(FILES lang/de/tqslapp.mo DESTINATION share/locale/de/LC_MESSAGES)
 	install(FILES lang/de/wxstd.mo DESTINATION share/locale/de/LC_MESSAGES)
